src/common/chromatic_adaptation.h

```diff
@@ -344,10 +344,7 @@ static inline void XYZ_adapt_D50(const dt_aligned_pixel_t lms_in,
 
   // Precomputed D50 primaries in XYZ for camera WB adjustment
   const dt_aligned_pixel_t D50 = { 0.9642119944211994f, 1.0f, 0.8251882845188288f, 0.f };
-
-  lms_out[0] = lms_in[0] * D50[0] / origin_illuminant[0];
-  lms_out[1] = lms_in[1] * D50[1] / origin_illuminant[1];
-  lms_out[2] = lms_in[2] * D50[2] / origin_illuminant[2];
+  for_each_channel(c) lms_out[c] = lms_in[0] * D50[c] / origin_illuminant[c];
```

Comment:
Should be `lms_in[c]`

Reply:
Oops, indeed. Thanks, should be fixed now.